{"links": {"self": {"href": "data/repositories/philipstarkey/qtutils/pullrequests/2/comments/43159146.json"}, "html": {"href": "#!/philipstarkey/qtutils/pull-requests/2/_/diff#comment-43159146"}}, "parent": {"id": 43159113, "links": {"self": {"href": "data/repositories/philipstarkey/qtutils/pullrequests/2/comments/43159113.json"}, "html": {"href": "#!/philipstarkey/qtutils/pull-requests/2/_/diff#comment-43159113"}}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 2, "links": {"self": {"href": "data/repositories/philipstarkey/qtutils/pullrequests/2.json"}, "html": {"href": "#!/philipstarkey/qtutils/pull-requests/2"}}, "title": "Python 3"}, "content": {"raw": "I'm running it from a unrelated directory with \"python -m blacs\". The import that causes problems is the one from the tabs and they get initialized after the directory is changed.", "markup": "markdown", "html": "<p>I'm running it from a unrelated directory with \"python -m blacs\". The import that causes problems is the one from the tabs and they get initialized after the directory is changed.</p>", "type": "rendered"}, "created_on": "2017-08-20T19:04:17.589631+00:00", "user": {"display_name": "Jan Werkmann", "uuid": "{44c4905c-2b90-4045-a5f1-652b8e228626}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B44c4905c-2b90-4045-a5f1-652b8e228626%7D"}, "html": {"href": "https://bitbucket.org/%7B44c4905c-2b90-4045-a5f1-652b8e228626%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:a70cc9cf-684e-4849-a61a-9ade4d7218b5/07e5095a-4741-4dc0-a462-9c7d455f961d/128"}}, "nickname": "PhyNerd", "type": "user", "account_id": "557058:a70cc9cf-684e-4849-a61a-9ade4d7218b5"}, "updated_on": "2017-08-20T19:04:17.629109+00:00", "type": "pullrequest_comment", "id": 43159146}